llvm/test/tools/sycl-post-link/sycl-properties-alignment-loadstore.ll

```diff
@@ -0,0 +1,64 @@
+; RUN: sycl-post-link --device-globals --ir-output-only -S %s -o %t.ll
+; RUN: FileCheck %s -input-file=%t.ll
+;
+; TODO: Remove --device-globals once other features start using compile-time
+;       properties.
+;
+; Tests the translation of "sycl-kernel-arg-attribute" to "spirv.ParameterDecorations" metadata
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
```

Comment:
I am not sure if we need the target datalayout and target triple here. Please check other tests in the same location. I am ok with it being here.

Reply:
done